web-client/src/views/TrialSessionWorkingCopy/WorkingCopyFilterHeader.tsx

```diff
@@ -10,13 +10,23 @@ export const WorkingCopyFilterHeader = connect(
     filters: state.trialSessionWorkingCopy.filters,
     trialSessionWorkingCopyHelper: state.trialSessionWorkingCopyHelper,
     trialStatusFilters: state.trialSessionWorkingCopyHelper.trialStatusFilters,
+    trialSessionWorkingCopy: state.trialSessionWorkingCopy,
   },
   function WorkingCopyFilterHeader({
     autoSaveTrialSessionWorkingCopySequence,
     filters = {},
     trialSessionWorkingCopyHelper,
     trialStatusFilters,
+    trialSessionWorkingCopy,
   }) {
+    const trialStatusCounts = Object.values(trialSessionWorkingCopy.caseMetadata).reduce((counters, c) => {
```

Comment:
Similar comment to above, with the added comment that this feels duplicative. Can we consolidate these two aggregations somehow? (I realize that the whole components are duplicative, but no reason to avoid adding salt to the wound!)